bot.py

```diff
@@ -27,7 +27,6 @@ def log(self, message):
     def close(self):
         self.file.close()
 
-
 class MusicBot(irc.IRCClient):
```

Comment:
Yeah, there are more python conventions... Which aren't being  used.
For example; https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables